yarn-project/end-to-end/src/e2e_fees/private_payments.test.ts

```diff
@@ -118,27 +118,27 @@ describe('e2e_fees private_payment', () => {
 
     const tx = await interaction.send().wait();
 
-    await expect(t.getCoinbaseBalance()).resolves.toEqual(InitialSequencerL1Gas + 1n);
-
     /**
      * at present the user is paying DA gas for:
      * 3 nullifiers = 3 * DA_BYTES_PER_FIELD * DA_GAS_PER_BYTE = 3 * 32 * 16 = 1536 DA gas
      * 2 note hashes =  2 * DA_BYTES_PER_FIELD * DA_GAS_PER_BYTE = 2 * 32 * 16 = 1024 DA gas
-     * 964 bytes of logs = 964 * DA_GAS_PER_BYTE = 964 * 16 = 15424 DA gas
+     * 348 bytes of logs = 348 * DA_GAS_PER_BYTE = 348 * 16 = 5568 DA gas
      * tx overhead of 512 DA gas
-     * for a total of 18496 DA gas.
+     * for a total of 12928 DA gas.
      *
      * The default teardown gas allocation at present is
      * 100_000_000 for both DA and L2 gas.
      *
-     * That produces a grand total of 200018496n.
+     * That produces a grand total of 200012928n.
      *
      * This will change because:
      * 1. Gas use during public execution is not currently incorporated
-     * 2. We are presently squashing notes/nullifiers across non/revertible during private exeuction,
+     * 2. We are presently squashing notes/nullifiers across non/revertible during private execution,
      *    but we shouldn't.
      */
-    expect(tx.transactionFee).toEqual(200018496n);
+
+    expect(tx.transactionFee).toEqual(200012928n);
```

Comment:
This was updated after recent work on logs, but wasn't caught because the test was not run as part of CI.